pages/s/[org]/[repo]/contributors.tsx

```diff
@@ -146,8 +146,10 @@ export default function RepoPageContributorsTab({ repoData, ogImageUrl }: RepoPa
             <TabList tabList={tabList} selectedTab={"contributors"} pageId={`/s/${repoData.full_name}`} />
           </div>
           <ClientOnly>
-            <Activity repositories={[repoData.id]} />
-            <Contributors repositories={[repoData.id]} defaultLayout="grid" />
+            <div className="flex flex-col gap-8">
```

Comment:
nit: prefer grid for this use case.

```suggestion
<div className="grid gap-8">
```